ui/components/ui/card/README.mdx

```diff
@@ -0,0 +1,42 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
```

Comment:
Is there anyway to prevent this from showing in the markdown?

Reply:
I don't think so... This is the format storybook uses to enable showing components inside of markdown. But yeah outside of storybook the developer experience isn't great